client/keys/utils.go

```diff
@@ -83,9 +84,9 @@ func ReadPassphraseFromStdin(name string) (string, error) {
 }
 
 // initialize a keybase based on the configuration
-func GetKeyBaseFromDir(rootDir string) (keys.Keybase, error) {
+func GetKeyBaseFromDir(rootDir string, readonly bool) (keys.Keybase, error) {
```

Comment:
my humble proposal:

set `readonly` to true by default to increase security. and make a separate method `GetKeyBaseFromDirWithWritePerm(rootDir string)` and call it where needed.

Reply:
I like this proposal. Do you mind modifying this @ackratos ?

Reply:
will do that thanks @melekes , glad to see you here:)

Reply:
fixed @melekes @jackzampolin